app/assets/stylesheets/main.scss

```diff
@@ -81,3 +81,12 @@
     margin-bottom: 0.5rem;
     margin-top: 0;
   }
+
+  .copy-button {
+    margin-left: 10px;
+  }
+
+  .copy-nowrap {
+    white-space: nowrap;
+    overflow: hidden;
```

Comment:
Downside of this approach to keeping the icon tied to the URL (and not getting wrapped to another line) is that it can cause overlapping tables if you make your screen too narrow. Maybe not a concern for a desktop-oriented staff app, but if there's another way to address this need, happy to tweak.